bin/grocery-delivery

```diff
@@ -66,8 +66,15 @@ def read_checkpoint
 end
 
 def full_upload(knife)
-  GroceryDelivery::Log.warn('Uploading all cookbooks')
-  knife.cookbook_upload_all
+  if GroceryDelivery::Config.berks
+    GroceryDelivery::Log.warn('Uploading all cookbooks with berkshelf')
+    GroceryDelivery::Log.debug(
```

Comment:
style nit:
* If the opening paren gets it's own line so does the closing
* indentation is two spaces:

```
    GroceryDelivery::Log.debug(
      "using cookbook paths: #{GroceryDelivery::Config.cookbook_paths}"
    )
    knife.berks_cookbook_upload_all
  else
```